src/compiler/commandLineParser.ts

```diff
@@ -312,9 +312,19 @@ namespace ts {
             }
         },
         {
-            name: "traceModuleResolution",

```

Comment:
this flag was planned to initially appear in 2.0 which was not officially released yes so no, technically it is not a breaking change. There are few folks who might already use this flag, I'll ping them once this PR is approved. Re experimental - I don't think this is applicable here.